src/MCPServer/lib/assets/workflow.json

```diff
@@ -7391,6 +7564,25 @@
         "sv": "Återingestera AIP"
       }
     },
+    "856d2d65-cd25-49fa-8da9-cabb78292894": {
+      "config": {
+        "@manager": "linkTaskManagerChoice",
+        "@model": "MicroServiceChainChoice",
+        "chain_choices": [
+          "6e431096-c403-4cbf-a59a-a26e86be54a8",
+          "63767e4b-9ce8-4fe2-8724-65cc1f763de0"
+        ]
+      },
+      "description": {
+        "en": "Do you want to scan for viruses in metadata?"
+      },
+      "exit_codes": {},
+      "fallback_job_status": "Failed",
+      "fallback_link_id": null,
+      "group": {
+        "en": "Process metadata directory"
```

Comment:
Should we include the rest of existing translations (e.g. `es`, `no`, etc) for the `Process metadata directory` group here?